PacketDefinitions420/PacketDefinitions/S2C/SpawnMinion.cs

```diff
@@ -4,33 +4,33 @@
 
 namespace PacketDefinitions420.PacketDefinitions.S2C
 {
-    public class SpawnPlaceable : Packet
+    public class SpawnMinion : Packet
     {
-        public SpawnPlaceable(IPlaceable p)
+        public SpawnMinion(IMinion m)
```

Comment:
I think you can just delete this file and use LeaguePackets version. this version of 4.20 is not actually correct, and not used (after spawn placeable change).